crates/symbolicator/src/services/objects/data_cache.rs

```diff
@@ -424,10 +475,17 @@ mod tests {
                 ..find_object
             };
             let result = objects_actor.find(find_object.clone()).await.unwrap();
-            assert_eq!(result.meta.unwrap().status, CacheStatus::Malformed);
+            assert_eq!(
+                result.meta.unwrap().status,
+                CacheStatus::Malformed(MalformedCause::Timeout)
+            );
+            // Timeouts don't retry ):
```

Comment:
I think you can tweak the per-connection vs per-request timeouts to trigger a retry here